core/trino-spi/src/main/java/io/trino/spi/StandardErrorCode.java

```diff
@@ -153,6 +153,7 @@ public enum StandardErrorCode
     INVALID_PLAN(129, USER_ERROR),
     INVALID_VIEW_PROPERTY(130, USER_ERROR),
     INVALID_ENTITY_KIND(131, USER_ERROR),
+    BAD_DATA(132, USER_ERROR),
```

Comment:
We should use `HIVE_BAD_DATA` instead of adding a global error code

Reply:
`HIVE_BAD_DATA` code it can't be used in the `trino-hive-formats` package since it comes from the `trino-hive` module., which in turn depends on `trino-hive-formats`.